Roommatch/Roommatch/ViewControllers/CreateProfileViewController.m

```diff
@@ -0,0 +1,31 @@
+//
+//  CreateProfileViewController.m
+//  Roommatch
+//
+//  Created by Lily Yang on 7/7/22.
+//
+
+#import "CreateProfileViewController.h"
+
+@interface CreateProfileViewController ()
+
+@end
+
+@implementation CreateProfileViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```

Comment:
Before submitting remove any code you don't want reviewed or you don't want to be in your app.